index.html

```diff
@@ -4,14 +4,34 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="assets/css/style.css">
     <script type="module" src="assets/js/game.js" defer></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/howler/2.2.3/howler.min.js" integrity="sha512-6+YN/9o9BWrk6wSfGxQGpt3EUK6XeHi6yeHV+TYD2GR0Sj/cggRpXr1BrAQf0as6XslxomMUxXp2vIl+fv0QRA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
 
     <title>emojiman</title>
 </head>
+<div id="intro">
+    <p id="intro-text">VAMPIRE PARTY Baddies: 🧛 Collect: 🫀🩸🧄 🌞 20 points each Destination : ⚰️
```

Comment:
could we do a id="intro-title" and id="intro-text" beneath it?  That will match up with what Andy is doing in his map JSON files.

Reply:
looks like a good base for now, but we might need different elements with separate ids for
- title
- text
- baddies
- points
- goal/aim